### PR TITLE
pin el7 acceptance tests to centos 7.6.1810 image

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

**Comment by mmoll** [comment]
still failing :broken_heart:

File: .sync.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 .travis.yml:
   beaker_sets:
     - centos6-64
+    - 'centos7-64{image=centos:7.6.1810}'
```
Comment (ekohl) [inline .sync.yml:8]: This specific syntax isn't going to work with our modulesync template. https://github.com/theforeman/foreman-installer-modulesync/pull/121 is intended to take care of this.

File: .travis.yml [hunk 2]
```diff
@@ -31,6 +31,28 @@ matrix:
         - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
         - sudo service docker restart
 
+    - rvm: 2.5.1
+      env:
+        - BEAKER_PUPPET_COLLECTION=puppet5
+        - BEAKER_setfile=centos7-64{hostname=centos7-64.example.com,image=centos:7.6.1810}
```
Comment (ekohl) [inline .travis.yml:37]: I think this is being interpreted by bash as something to expand, causing things to fail.

**Comment by ekohl** [comment]
This was done a while back when we migrated to GHA.

**Pull request closed without merging**